modules/conversion-center/assets/scss/widgets/contact-buttons-editor.scss

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
This should be placed in `modules/conversion-center/assets/scss/editor/contact-buttons-editor.scss`